data/boarding missions.txt

```diff
@@ -37,3 +37,31 @@ mission "Assisting Hai"
 			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
 				decline
 	destination Hai-home
+
+mission "Navy emergency"
+	assisting
+	repeat
+	deadline
+	passengers 1 3
+	description "Deliver <fare> to <destination> by <date>: the military hospital is waiting for you."
+	to offer
+		"reputation: Navy (Oathkeeper)" > 100
+		"passenger space" > <bunks>
+		random < 15
+	source
+		government "Navy (Oathkeeper)"
+		system "Arneb" "Hatysa" "Alnilam" "Unagi" "Almaaz" "Mintaka" "Gorvi" "Tortor" "Fah Root" "Rigel" "Hassaleh" "Danoa" "Rajak" "Sumar" "Ultima Thule" "Cardax" "Volax" "Moktar"
```

Comment:
You could also use the `near` argument.

Reply:
In addition to near, there are `neighbor` and `not` filters that you may find useful. Planet and system attributes may be additionally useful.

Reply:
Farpoint could be the planet equipped to deal with the injured (it is also the base for the Oathkeepers).